client/src/components/BlogList/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+import axiosConfig from '../../config/axiosConfig';
+import BlogCard from '../BlogCard';
+import LoaderCentered from '../../elements/LoaderCentered/LoaderCenter';
+
+export default function BlogList() {
+  const [blogs, setBlogs] = useState([]);
+
+  const fetchBlogs = async () => {
+    const params = {
+      limit: 10,
+      skip: 0,
+      orderBy: 'createdAt',
+      orderType: -1,
+    };
+    const response = await axios.get(`posts?limit=${params.limit}&skip=${params.skip}&orderby=${params.orderBy}&orderType=${params.orderType}`, axiosConfig);
+    setBlogs(response.data.posts);
+  };
+
+  useEffect(() => {
+    fetchBlogs();
+  }, []);
+
+  const conditionallyRenderBlogs = () => {
+    if (blogs.length === 0) {
+      return <LoaderCentered />;
+    }
+    return blogs.map((blog) => {
+      const refactoredData = {
+        ...blog,
+        content: blog.content || 'Some description...',
+        image: blog.image || 'https://source.unsplash.com/aJTiW00qqtI',
```

Comment:
create a config file to get common data for each environment, do not hard code image path

Reply:
we'll have same data for each environment, so no need for a config file, will create a new file for constants though

Reply:
even though we have same data for each environment, urls can change, so its better to have a config file, that takes environment variables from netlify.